### ISSUE TITLE
Doesn't work on Cray XC50 system with more than two nodes([nid00625:22636] [[53334,0],0] ORTE_ERROR_LOG: Not found in file plm_alps_module.c at line 423 )

### ISSUE THREAD

**Issue by ys-r** [open-mpi/ompi#8670]
### What version of Open MPI are you using? (e.g., v3.0.5, v4.0.2, git branch name and hash, etc.)
v4.0.3
v4.1.0

### Describe how Open MPI was installed (e.g., from a source/distribution tarball, from a git clone, from an operating system distribution package, etc.)
from a source

### Please describe the system on which you are running

* Operating system/version: Cray XC50 SUSE Linux Enterprise Server 12 SP3  (x86_64) - Kernel
* Computer hardware: 2cpus per node(36cores per node)
* Network type: Cray Aries (uGNI)

-----------------------------

## Details of the problem
Hello, I have compiled and used OpenMPI in OpenFOAMv2006.
The makefile is attached. [makeOPENMPI.txt](https://github.com/open-mpi/ompi/files/6179637/makeOPENMPI.txt)
The following sample code works under 36 parallel, and the execution result is as follows.
[mpihello.c.txt](https://github.com/open-mpi/ompi/files/6179653/mpihello.c.txt)

```job.jcl
#PBS -l mppwidth=36

module load ccm
ccmrun PATH_TO_OMPI/bin/mpiexec --hostfile $HOME/.crayccm/ccm_nodelist.${PBS_JOBID} ./a.out
```

```result
Hello world from processor nid00625, rank 18 out of 36 processors
...
Hello world from processor nid00625, rank 33 out of 36 processors
```
Normally, on XC50 systems, programs are started using aprun. However, CRAY support instructed us to use ccmrun.
Either way, it's working fine.
 However, it did not work on 37 or more parallel systems because of the following error.

```job.jcl
#PBS -l mppwidth=72
...
module load ccm
ccmrun PATH_TO_OMPI/bin/mpiexec --hostfile $HOME/.crayccm/ccm_nodelist.${PBS_JOBID} ./a.out

```

```result
[nid00625:22636] [[53334,0],0] ORTE_ERROR_LOG: Not found in file plm_alps_module.c at line 423
```

I tried the following, but a same error occured:
openmpi-4.1.0 build without any options ( without make using 'makeOpenMPI' ). 

Thanks.

**Comment by hppritcha** [comment]
I don't think you're going to make much progress using cray ccmrun with Open MPI.  What happens if you use the mpirun that gets installed with Open MPI?

**Comment by ys-r** [comment]
It works well!　I also checked the operation of the communication using example/connectivity_c.c, then it also works.
I'm checking with CRAY support staff if mpirun is running correctly in the computation node, not in the login node.

I did not doubt what the CRAY support staff said.
OpenFOAM still doesn't work well in parallel processing, but I don't think OpenMPI is the cause of this.

Thank you.

**Comment by rhc54** [comment]
OMPI won't run on the login node as the login node isn't included in the allocation. You should be just fine. Let us know if you hit any further problems!

**Comment by hppritcha** [comment]
@ys-r is this issue resolved for you?

**Comment by ys-r** [comment]
> @ys-r is this issue resolved for you?

Yes. This error will no longer occur.

However, OpenFOAM still doesn't work beyond a certain number of parallelism.
Whether the problem is with OpenFOAM or OpenMPI is not clear to me. 
If I identify the problem, I will create a new issue.

Thanks.
